examples/crpc-common.mjs

```diff
@@ -3,14 +3,11 @@ import chalk from 'chalk';
 export const mkAssertEq = rpc => async (l, e, a) => {
   const f = x => JSON.stringify(x).replace(/\\u0000/g, '');
   const [ ej, aj ] = [ f(e), f(a) ];
-  if (ej === aj) {
+  if (ej === aj)
```

Comment:
I hate `if`s and `elses` without `{}`. I have made the mistake so many times of adding an expression